rust-server/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "rust-libp2p-webrtc-server"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0"
+futures = "0.3.27"
+libp2p = {version="0.51.1", features = ["tokio", "gossipsub", "webrtc", "macros", "kad"]}
```

```suggestion
libp2p = {version="0.51.1", features = ["tokio", "gossipsub", "webrtc", "macros", "kad", "rsa", "ed25519"]}
```

The former for compatibility with implementations that use `rsa`, the latter given that `ed25519` will soon no longer be a default option.